### ISSUE TITLE
Feature Request: Add Organization Team Auto Join Option

### ISSUE THREAD

**Issue by kimberly-eis** [go-gitea/gitea#10510]
- Gitea version (or commit ref): 1.11
- Git version: 
- Operating system:
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:

## Description
I would like the ability for a public organization to have a team that a user can self add themselves to.
...

**Comment by 6543** [comment]
yes something like an option where you can set a team as auto-join destination and if a organisation has set this option autojion is enabled ...
